packages/angular_devkit/build_angular/src/webpack/configs/browser.ts

```diff
@@ -47,6 +47,7 @@ export function getBrowserConfig(wco: WebpackConfigOptions): webpack.Configurati
       },
       perChunkOutput: false,
       outputFilename: '3rdpartylicenses.txt',
+      skipChildCompilers: true,
```

Comment:
This option might cause licenses not to be extracted
https://github.com/xz64/license-webpack-plugin/issues/83

Reply:
The license plugin was also added directly to the web worker compilation (the other location where code is compiled) to allow capture of licenses.  If `perChunkOutput` is set to false in combination with child compilations (including internal ones such as in the CSS extraction), the plugin becomes incompatible with Webpack 5 since the duplicate asset warning in Webpack 4 is now an error.

Alternatively, we could only change the outputFilename to `3rdpartylicenses-[hash].txt` which will generate a unique file per compilation including each child compilation.  Downside is that potentially thousands of these will be created (one per component stylesheet) all with `css-loader` as a dependency even though they are extracted to CSS and have no code.

Reply:
Let’s keep it like this for now. We can iterate in the future if needs be.